iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/AnonymousAccessToResourceCheck.java

```diff
@@ -100,6 +104,22 @@ private static void checkApiManagementServiceApis(ContextualResource resource) {
       .reportIfAbsent(APIMGMT_AUTHENTICATION_SETTINGS_NOT_SET_MESSAGE);
   }
 
+  private static void checkStorageAccounts(ContextualResource resource) {
+    resource.property("allowBlobPublicAccess")
+      .reportIf(CheckUtils.isTrue(), STORAGE_ANONYMOUS_ACCESS_MESSAGE)
+      .reportIfAbsent(STORAGE_ANONYMOUS_ACCESS_MESSAGE);
+
+    ContextualResource containers = resource.childResourceBy("blobServices/containers", r -> true);
+    if (containers.isPresent()) {
+      checkStorageAccountContainers(containers);
+    }
```

Comment:
I'm not sure if this is needed. Nested resources should be handled automatically. If not please create ticket.

Reply:
Indeed we process child resources already; I missed this part in `AbstractArmResourceCheck` on the first look. Thanks for the heads-up!